CHANGELOG.md

```diff
@@ -15,7 +15,7 @@ master
 * Update Rubocop and apply new lint rules
 * Do not create unnecessary array in IgnoreDescriptor (#2183)
 * Fix reload of watched sources with destination_dir (#2190)
-* Fix localisation and recursion issues of Traversal::parent. (#2188)
+* Fix localization and recursion issues of Traversal::parent. (#2188)
```

Comment:
While your repo is mostly en-gb, the majority uses localiz*, so I'm applying that spelling.

Reply:
Seems good. Probably came from a en-gb speaking PR title